jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -313,17 +313,27 @@ private static boolean isSamePackage(Class<?> a, Class<?> b){
 				return true;
 			}
 			
+			Class<?> classA = a;
```

Comment:
Why introduce `classA` here?  Especially when the loop doesn't use it....

Reply:
I prefer not to modify method arguments, but I can remove the locals if you wish.  What is the convention?

Reply:
I'm not aware of any convention that says not to update arguments

Reply:
Fair enough.  I removed the auxiliary local variable.